hrm-service/src/contact-job/contact-job-complete.ts

```diff
@@ -1,4 +1,8 @@
-import { completeContactJob, ContactJobType } from './contact-job-data-access';
+import {
+  ContactJobType,
+  completeContactJob,
+  appendFailedAttemptPayload,
+} from './contact-job-data-access';
 import { deletedCompletedContactJobs, pollCompletedContactJobs } from './client-sqs';
```

Comment:
Might be nice to alias these methods to something that mentions the queue, e.g., `deleteCompletedContactJobsFromQueue`, to prevent confusing them for DB operations